server/src/com/cloud/network/router/VirtualNetworkApplianceManagerImpl.java

```diff
@@ -1588,6 +1588,16 @@ protected StringBuilder createGuestBootLoadArgs(final NicProfile guestNic, final
         return buf;
     }
 
+    /**
+     * Return a string representing network Cidr for the specifeid network
+     * @param guestNetwork
+     * @return valid network Cidr for the specified network
+     */
+    protected String getValidNetworkCidr(Network guestNetwork){
```

Comment:
I think you should extract this method to another common class (e.g. NetworkModel) since this method can be reused over the different classes.